.gitignore

```diff
@@ -24,3 +24,4 @@ node_modules
 __pycache__
 *.swp
 .vscode/
+.idea/
```

Comment:
I added it because my IDE (PyCharm) keeps generating project setting files under .idea folder. I think it would free me from deleting it every time I push lol.